dds_cli/utils.py

```diff
@@ -147,7 +147,7 @@ def request_get(
     headers,
     params=None,
     json=None,
-    error_message="API Request failed.",
+    error_message=None,
```

Comment:
We need to have some kind of default or require the user to input it (remove `=None`), otherwise there's a risk it will display `None: API Request failed` for example.

Reply:
Yes indeed, I wanted to have it that way but I managed to fool myself that it should be None